src/boost_fractor.jl

```diff
@@ -66,10 +68,71 @@ mutable struct SetupBoundaries
     eps::Array{Complex{Float64},1}   # = [1,9,1]
     relative_tilt_x # = [0,0]
     relative_tilt_y # = [0,0]
-    relative_surfaces::Array{Complex{Float64},3} # = [z, x,y ]
+    relative_surfaces::Array{Complex{Float64},3} # = [z, x, y]
     # etc.
 end
 
+struct Phase_shifts
+    #pre-calculate k0 for each boundary.
+    k0::Array{ComplexF64, 1}
+
+    k_prop::Array{ComplexF64, 3}
+    #combine surface misalignments and tilts into one array of phase-shifts
+    surface::Array{ComplexF64, 3}
+    #surface misalignments can usually be applied for only the area within the diskradius
+    #for dancer and cheerleader: need to account for the first/last boundaries.
+    #single 2D slice for dancer, two for cheerleader, unused for transformer
+    surface_out::Array{ComplexF64,3}
+    #two more values that are needed
+    diskR::Float64
+    dz::Array{Float64,1}
+    #convenient arrays for slicing to get the subarray enclosing the disk area
+    x_sub::Array{Int64, 1}
+    y_sub::Array{Int64, 1}
+end
+
+#calcualte the phase shifts for every boundary
+function init_prop(bdry::SetupBoundaries, coords::CoordinateSystem, lambda, radius; output_surfaces = [])
```

Comment:
Phase_shifts and init_prop seem to break naming conventions?
If I understand correctly they should be named PhaseShifts and SeedPhaseShifts to follow the same structure as the other initializing functions.
Please also add documentation to the init_prop function (the same way the documentation is done for existing funcs).
Also maybe not squeeze the two functions in between SetupBoundaries and SeedSetupBoundaries, but after both?